private/pkg/licenseheader/licenseheader.go

```diff
@@ -84,6 +84,9 @@ All rights reserved.`,
 		".kt":    "//",
 		".swift": "//",
 		".java":  "//",
+		".cc":    "//",
+		".c":     "//",
+		".h":     "///",
```

Comment:
Is this right? Just making sure

Reply:
@Alfus I was going off of protovalidate-cc's `.h` files. Is the triple slash the way to go?

Reply:
should be double slash, triple is for doc strings